src/multibody/model.hpp

```diff
@@ -503,7 +503,7 @@ namespace se3
     ///
     /// \param[in] model The model structure of the rigid body system.
     ///
-    Data (const Model & model);
+    explicit Data (const Model & model);
```

Comment:
:+1:
As discussed with @nmansard, the majority of the constructors must be explicit either in multi body or in spatial classes